bin/command.php

```diff
@@ -561,7 +564,11 @@ private static function parse_docblock( $docblock ) {
 		}
 		$ret['description'] = str_replace( '\/', '/', trim( $ret['description'], PHP_EOL ) );
 		$bits = explode( PHP_EOL, $ret['description'] );
-		$ret['short_description'] = array_shift( $bits );
+		$short_desc = array( array_shift( $bits ) );
+		if ( !empty( $bits[0] ) ) {
```

Comment:
This fixes the code so that it works for 2 lines of short description. But it would break again for 3 lines or more.

We should build this in such a way that it works consistently with any number of lines, even if it is debatable whether we really need a short description that spans a dozen lines.